hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -712,7 +717,17 @@ public void close() {
     }
 
     /**
-     * Clean up any stale/old files/data lying around (either on file storage or index storage)
+     * Clean up any stale/old files/data lying around (either on file storage or index storage) that is past
+     * the typical query timeout. Default is 12 hours.
+     */
+    public void clean() throws HoodieIOException {
+        String startCleanTime = startCommit();
+        clean(startCleanTime);
+    }
+
+    /**
+     * Clean up any stale/old files/data lying around (either on file storage or index storage) that is past
+     * the typical query timeout. Default is 12 hours.
```

Comment:
>> given cleaner itself has these knobs
True. I just added more description on the API as we are making this public. I will add that based on the cleaning policy. 

>> Same comment on index tagLocation and updateLocation...
I suppose adding metrics? That can be a seperate diff.